framework-plugins/lisk-framework-forger-plugin/src/controllers/forging.ts

```diff
@@ -42,8 +58,14 @@ interface ForgingResponseData {
 
 interface ForgingRequestData extends ForgingResponseData {
 	readonly password: string;
+	readonly height: number;
+	readonly maxHeightPreviouslyForged: number;
+	readonly maxHeightPrevoted: number;
+	readonly overwrite?: boolean;
 }
 
+const isGreaterThanZero = (value: number) => value === null || value === undefined || value < 0;
```

Comment:
the input should be either `unknown`, `value? number`, or `value: number | null | undefined`?
Also, function name sounds opposite to me